README.rst

```diff
@@ -277,6 +277,34 @@ of the (type of) catalog object they once annotated:
 0
 
 
+Atomic operations on metadata
+-----------------------------
+
+With release 4.2.8 of iRODS, the atomic metadata API was introduced to allow a group of metadata add and remove
+operations to be performed transactionally, within a single call to the server.  This capability can be leveraged in
+version 0.8.6 of the PRC.
+
+So, for example, if 'obj' is a handle to an object in the iRODS catalog, we can send an arbitrary number of AVUOperation
```

Comment:
please additionally note that this works for objects, collections, users, and resources.